docs/production-deployment/cloud/nexus/observability.mdx

```diff
@@ -0,0 +1,50 @@
+---
+id: observability
+slug: /cloud/nexus/observability
+title: Obserability - Temporal Nexus
+description: Learn about integrated observability in Temporal Nexus including cloud metrics and audit log streaming.
+sidebar_label: Observability
+tags:
+  - Temporal Cloud
+  - Nexus
+keywords:
+  - Temporal Nexus
+  - Nexus
+  - observability
+  - metrics
+  - cloud metrics
+  - audit log
+  - audit log streaming
+---
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+Temporal Nexus is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
+Learn why you should use Nexus in the [evaluation guide](/evaluate/nexus).
+
+:::
+
+Nexus provides Cloud Metrics and Audit Log streaming, in addition to integrated [execution debugging](/nexus/execution-debugging).
+
+## Metrics
```

Comment:
I would direct users to the `sdk-metrics` page and include a summary of the Nexus related metrics there.
Using gRPC request metrics isn't ideal.